package.json

```diff
@@ -1,10 +1,16 @@
 {
   "name": "angular-sortablejs",
-  "version": "1.2.1",
+  "version": "1.3.0",
   "description": "SortableJS for Angular 2+",
-  "main": "index.js",
+  "main": "./dist/index.js",
+  "typings": "./dist/index",
   "scripts": {
+    "clean": "rimraf ./dist/* ./codegen/*",
     "build": "tsc",
+    "prebuild": "npm run clean",
+    "build-dist": "ngc -p tsconfig-dist.json",
```

Comment:
Yep, definitely. It's still there if one contributor wants to try with a development, JIT-enabled build, maybe because something gets broken only in AOT build. It does not harm if it's there, and it can be useful to leave a *dev mode* set of scripts as well.